tools/travis-before-install.sh

```diff
@@ -4,6 +4,15 @@ uname -a
 free -m
 df -h
 ulimit -a
+
+if [ -n "$PPC64_LE" ]; then
+  pwd
+  ls -ltrh
```

Comment:
could remove these two lines -- debug mostly -- relocation to fix path issue relative to master